app/assets/stylesheets/index.scss

```diff
@@ -6,7 +6,7 @@ $black: #000000;
 $white: #ffffff;
 
 body {
-  background-image: image-url("new_year_background.png");
+  background-image: url('<%= asset_path("new_year_background.jpg") %>');
```

Comment:
Invalid URL `<%= asset_path("new_year_background.jpg") %>`: bad URI(is not URI?): <%= asset_path("new_year_background.jpg") %>